README.md

```diff
@@ -0,0 +1,111 @@
+[![Build Status](https://travis-ci.org/mortheria/dasher.svg?branch=master)](https://travis-ci.org/mortheria/dasher)
```

Comment:
There are many subprojects that we will add build status for. Therefore, prefix this with "Linux:".